resources/views/models/blog/articles/form.blade.php

```diff
@@ -0,0 +1,124 @@
+
```

Comment:
Do you mind fixing the indentation for this whole file. I believe PhpStorm can help with a keyboard shortcut `OPTION+CMD+L`